karmagrambot/__init__.py

```diff
@@ -8,13 +8,16 @@
 
 logging.basicConfig()
 
+def already_voted(replied, user_id):
```

Comment:
Please add a docstring to new functions.

Let's adopt Google style for docstrings:

```
<Short description>

Args:
    arg_name: <description>

Returns:
    <description>
```

Reply:
Done